### PR TITLE
Feat/private profile

### PR DESCRIPTION
**Pull request by donovanCarreno**

### CODE REVIEW

**Comment by natgrigorovskaya** [comment]
⛵

File: client/src/PageLayout.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Nav from './components/HeaderFooter/Nav.jsx';
 import Foot from './components/HeaderFooter/Foot.jsx';
+import PrivateProfile from './components/Profile/PrivateProfile.jsx';
```
Comment (cassiebaer) [inline client/src/PageLayout.jsx]: Do we need this import?
\

File: client/src/components/HeaderFooter/Nav.jsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ export default class Nav extends React.Component {
           <div className="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
             <ul className="nav navbar-nav">
               <li><Link to="/">Categories</Link></li>
-              <li><Link to="/profile">My Profile</Link></li>
+              <li><Link to="/profile/1">My Profile</Link></li>
```
Comment (cassiebaer) [inline client/src/components/HeaderFooter/Nav.jsx]: This is hard-coded to userId=1

**Comment by cassiebaer** [comment]
👊

**Pull request merged**